_data/identity.yml

```diff
@@ -4,9 +4,10 @@ websites:
     img: 1password.png
     tfa:
       - totp
+      - u2f
       - hardware
     doc: https://support.1password.com/two-factor-authentication/
-    exception: "Teams using Duo with 1Password can't enable additional 2FA as it's already handled by Duo"
+    exception: "Teams using Duo with 1Password can't enable additional 2FA as it's already handled by Duo. Hardware tokens (not U2F) are only available through Duo."
```

Comment:
What are the non-U2F hardware tokens that Duo offers?

Reply:
From https://duo.com/product/multi-factor-authentication-mfa/authentication-methods/tokens-and-passcodes.

"Duo supports third-party hardware tokens, like Yubico’s YubiKeys, **or any OATH HOTP-compatible tokens**. Third-party hardware tokens can be imported into the system by an administrator.

**Duo’s D-100** tokens have an expected minimum battery lifetime of 2 years. They are only available in increments of 10. You can purchase tokens from the Duo Admin Panel or through your Account Team."

Reply:
From https://duo.com/product/multi-factor-authentication-mfa/authentication-methods/tokens-and-passcodes.

"Duo supports third-party hardware tokens, like Yubico’s YubiKeys, **or any OATH HOTP-compatible tokens**. Third-party hardware tokens can be imported into the system by an administrator.

**Duo’s D-100** tokens have an expected minimum battery lifetime of 2 years. They are only available in increments of 10. You can purchase tokens from the Duo Admin Panel or through your Account Team."